hack/Dockerfile

```diff
@@ -12,24 +12,55 @@ ENV PYTHONUNBUFFERED 1
 RUN set -ex \
 	&& apt-get update \
 	&& apt-get install -y --no-install-recommends \
+		atool \
 		apt-transport-https \
 		build-essential \
+		coreutils \
 		curl \
+		gcc \
 		gettext \
+		ghostscript \
 		git \
+		g++ \
 		gpg-agent \
+		hashdeep \
+		imagemagick \
+		inkscape \
 		libbz2-dev \
 		libffi-dev \
+		libldap2-dev \
 		liblzma-dev \
+		libimage-exiftool-perl \
+		libmysqlclient-dev \
 		libncursesw5-dev \
 		libreadline-dev \
+		libsasl2-dev \
 		libsqlite3-dev \
 		libssl-dev \
 		libxml2-dev \
 		libxmlsec1-dev \
+		libxslt1-dev \
 		locales \
+		logapp \
+		md5deep \
+		mediaconch \
+		mediainfo \
```

Comment:
Shouldn't these be placed **after** the `repo-mediaarea` package has been installed?